libraries/Microsoft.Bot.Builder.Azure/Storage/BlobsTranscriptStore.cs

```diff
@@ -0,0 +1,419 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Azure.Storage
+{
+    /// <summary>
+    /// The blobs transcript store stores transcripts in an Azure Blob container.
+    /// </summary>
+    /// <remarks>
+    /// Each activity is stored as json blob in structure of
+    /// container/{channelId]/{conversationId}/{Timestamp.ticks}-{activity.id}.json.
+    /// </remarks>
+    public class BlobsTranscriptStore : ITranscriptStore
+    {
+        private static readonly JsonSerializer JsonSerializer = JsonSerializer.Create(new JsonSerializerSettings
```

Comment:
>JsonSerializer [](start = 32, length = 14)

Same comments I made on the Blob class about json serializer.
Why 2, why static, why serialize type info.